test/unit/widgets/forms.test.ts

```diff
@@ -24,11 +23,9 @@ import {
   sortByLabel,
   sortBySequence
 } from '../../../src/widgets/forms'
-import * as ns from '../../../src/ns'
-jest.mock('rdflib')
-jest.mock('solid-auth-client')
-const dom = new JSDOM('<!DOCTYPE html><p>Hello world</p>').window.document
-const element = dom.createElement('div')
+
+// jest.mock('rdflib')
+// jest.mock('solid-auth-client')
```

Comment:
Why comment these out?

Reply:
I wasn't sure I needed them.  It works without, however I may need to add them back in as I keep adding tests.